server/prisma/migrations/20210630190517_init/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "email" TEXT NOT NULL,
+    "password" TEXT NOT NULL,
+    "firstName" TEXT,
```

Comment:
you shouldn't be camel casing DB columns. it's standard to `under_score_your_columns`.